bazel/v8.patch

```diff
@@ -319,12 +322,12 @@ index be6b72d157..a71968fd10 100644
 +      sizeof(T) == sizeof(double) && std::is_floating_point<T>::value;
 +
 +  static_assert(is_uint32 || is_uint64 || is_double);
- 
+
    const int target_align = 32;
    // Scalar loop to reach desired alignment
 @@ -256,21 +266,21 @@ TARGET_AVX2 inline uintptr_t fast_search_avx(T* array, uintptr_t array_len,
    }
- 
+
```

Comment:
Nit: This PR contains a dozen of unnecessary (whitespace) changes that make this much less readable, consider unstaging them.

Reply:
Yeah sorry, my editor seems too eager to remove "trailing" whitespaces.

Reply:
Much better now, thanks!